app/javascript/components/tenant-quota-form/helper.js

```diff
@@ -95,7 +97,7 @@ export const createRows = (initialValues, enforced, setEnforced, values, setValu
         },
       },
       Units: {
-        text: value.text_modifier,
+        text: __(value.text_modifier),
```

Comment:
This should be translated, ideally, on the API side, so the values come out of the API already translated.